src/gr11review/part2/Utility.java

```diff
@@ -1,9 +1,110 @@
 package gr11review.part2;
-import java.io.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
 
 public class Utility {
 
-    
+    //Method 1
+
+    public static String zipZap(String str){
```

Comment:
@amaan2106 all methods should have javadoc comments to describe purpose, parameters, and return value